Parsers/ASimNetworkSession/Parsers/imNetworkSession.yaml

```diff
@@ -1,126 +1,128 @@
-Parser:
```

Comment:
@jaimeesc Please undo these changes, difficult to validate the change this way. Please re-do so only SonicWall changes are clearly visible.

Reply:
I was really struck by this one too. I didn't change the full content of the file, but it replaces the entire file. I'll see what I can do to undo this.

Reply:
I tried restoring the file to its current master state, then updated it with SonicWall entries. I hope it looks ok this time.